app/controllers/AuthorisationsController.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.ukimauthcheckerapi.controllers
+
+import play.api.libs.functional.syntax.toFunctionalBuilderOps
+import play.api.libs.json._
+import play.api.mvc.{Action, ControllerComponents}
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+
+import java.time.LocalDate
+import javax.inject.{Inject, Singleton}
+
+@Singleton()
+class AuthorisationsController @Inject()(cc: ControllerComponents) extends BackendController(cc) {
+  def authorisations: Action[JsValue] = Action(parse.json) { request =>
+    val requestBody = request.body
+
+    val reads: Reads[(List[String], Option[LocalDate])] = (
+      (__ \ "eoris").read[List[String]] and
+        (__ \ "date").readNullable[String].map(_.map(LocalDate.parse))
+      ).tupled
+
+    val result = requestBody.validate(reads)
```

Comment:
Lets clean this up, it can be far cleaner and we don't need to force into a tuple.

Define a case class and object for the income json model (submission? check?) and put together an action that will parse or return 400 as an action and import that action for use here. I expect this controller would then basically look like the action is called and then we return 200.